sdk/python/tests/compiler/testdata/any_sequencer.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: any-sequencer
+  annotations:
+    tekton.dev/output_artifacts: '{}'
+    tekton.dev/input_artifacts: '{}'
+    tekton.dev/artifact_bucket: mlpipeline
+    tekton.dev/artifact_endpoint: minio-service.kubeflow:9000
+    tekton.dev/artifact_endpoint_scheme: http://
+    tekton.dev/artifact_items: '{"task1": [], "task2": [], "task3": [], "task4": []}'
+    sidecar.istio.io/inject: "false"
+    anyConditions: '{"any-sequencer-4b2c9": ["task1", "task2", "task3"]}'
+    pipelines.kubeflow.org/pipeline_spec: '{"description": "Any Sequencer Component
+      Demo", "name": "Any Sequencer"}'
+spec:
+  pipelineSpec:
+    tasks:
+    - name: task1
+      taskSpec:
+        steps:
+        - name: main
+          args: [sleep 15]
+          command: [/bin/bash, -c]
+          image: registry.access.redhat.com/ubi8/ubi-minimal
+      timeout: 0s
+    - name: task2
+      taskSpec:
+        steps:
+        - name: main
+          args: [sleep 200]
+          command: [/bin/bash, -c]
+          image: registry.access.redhat.com/ubi8/ubi-minimal
+      timeout: 0s
+    - name: task3
+      taskSpec:
+        steps:
+        - name: main
+          args: [sleep 300]
+          command: [/bin/bash, -c]
+          image: registry.access.redhat.com/ubi8/ubi-minimal
+      timeout: 0s
+    - name: task4
+      taskSpec:
+        steps:
+        - name: main
+          args: [sleep 30]
+          command: [/bin/bash, -c]
+          image: registry.access.redhat.com/ubi8/ubi-minimal
+      timeout: 0s
+      runAfter: [any-sequencer-4b2c9]
+    - name: any-sequencer-4b2c9
+      params:
+      - {name: pipelineRun-namespace, value: $(context.pipelineRun.namespace)}
+      - {name: pipelineRun-name, value: $(context.pipelineRun.name)}
+      taskSpec:
+        params:
+        - {name: pipelineRun-namespace}
+        - {name: pipelineRun-name}
+        steps:
+        - name: main
+          args: [-namespace, $(params.pipelineRun-namespace), -prName, $(params.pipelineRun-name),
+            -taskList, 'task1,task2,task3']
+          command: [any-taskrun]
+          image: dspipelines/any-sequencer:latest
```

Comment:
thanks @jinchihe, are you able to provide the source code for this `dspipelines/any-sequencer` image?

Reply:
Already commit in the PR, thanks @Tomcli